onebusaway-android/build.gradle

```diff
@@ -450,6 +451,10 @@ dependencies {
     implementation 'androidx.multidex:multidex:2.0.0'
     // Unit tests - seems like this is still necessary w/ Android X even though useLibrary is declared earlier
     androidTestImplementation 'androidx.test:runner:1.1.0'
+    // WorkManager (Java only)
+    implementation "android.arch.work:work-runtime:1.0.0-rc01"
+    // WorkManager - Kotlin + coroutines
```

Comment:
Let's remove the Kotlin + coroutines depedency if we're not using it.

Reply:
Removed the unused dependencies.